Makefile

```diff
@@ -65,16 +55,31 @@ record:
 record-flex:
 	@go run src/dividat-driver/recorder/main.go ws://localhost:8382/flex
 
+.PHONY: bin
+bin:
+	mkdir -p bin
+
+
+VERSION := $(shell git describe --tags --abbrev=0)
+
+.PHONY: VERSION
+VERSION:
+	echo $(VERSION) > VERSION
```

Comment:
Does this need to be a file instead of a value given as a parameter?

Reply:
Discussed here: https://github.com/dividat/driver/pull/129#discussion_r1554962503